Added missing civilian buildings and props

Mailaender opened the pull request:
Closes https://github.com/OpenRA/ra2/issues/48.

The rules https://github.com/OpenRA/OpenRA/pull/10714 and sequences https://github.com/OpenRA/OpenRA/pull/10715 were automatically converted and then manually cleaned to use inheritance.

I noticed a lot of wrongdoings in our current rule setup. Many were hard-coding one sprite variant which doesn't come with snow in the arctic tileset. Should be all fixed now. Also got rid of the crash when the repair crane gets damaged already detected by https://github.com/OpenRA/ra2/pull/108.

GraionDilach commented on rules/palettes.yaml:
```diff
@@ -8,6 +8,21 @@
 		Filename: unittem.pal
 		ShadowIndex: 1
+	PaletteFromFile@temperatecity:
+		Name: tsplayer
```
wrong?

Mailaender replied:
Yes, this was my first guess. Fixed.

abcdefg30 commented on rules/civilian-props.yaml:
```diff
@@ -102,29 +102,467 @@ camsc13:
+		Type: concrete
+	Health:
+		HP: 5
+	Building:
```
This shouldn't be necessary as it is inherited from `^CivBuilding`which inherits this from `^Building`.
(Same for the actors below that inherit `^CivBuilding`.)

abcdefg30 commented:
You can drop the `Armor: Type: wood` from all actors inheriting `^CivBuilding`, too.

I also think that the civilian vehicles would profit from a `^CivVehicle` definition which contains:
```
Inherits: ^Vehicle
Valued:
	Cost: 800
Mobile:
	Speed: 113
Health:
	HP: 100
Armor:
	Type: Light
RevealsShroud:
	Range: 5c0
Selectable:
	Bounds: 64, 52, 0, -6
RenderSprites:
RenderVoxels:
WithVoxelBody:
Voiced:
	VoiceSet: CarVoice
```

You can entirely drop `ROT: 5`, `RenderSprites:` and `RenderVoxels` as they are inherited, too.

I also saw some weird graphical glitches with the new buildings:
![mckongglitch](https://cloud.githubusercontent.com/assets/7704140/12895067/f7dfcc74-ce9a-11e5-9727-10e029990799.png)
It seems like that is the "husk" which should be left by the building (not necessarily in this pr ;)) which is rendered already for some reason (wrong `ShadowStart` definitions?).

I guess some time we also want to rework the tooltips of those `NY Building <number>` actors.

Anyway, sorry for causing inconvenience here, but I think our yaml will be much cleaner with the extra defaults. If you wish I can send a PR against your fork with the changes I want.

Other than that good job!

Mailaender commented:
Rebased, simplified rules and fixed some shadow sequences. Husks are indeed also already doable. Will do it in a followup as this is already pretty large.

abcdefg30 commented:
Looks good to me now. :+1: / :white_check_mark: 
Thanks.
